### ISSUE TITLE
RocketSimApp "MP4 Recording" shows only one finger touch when rotating a map.

### ISSUE THREAD

**Issue by roblabs** [AvdLee/RocketSimApp#240]
**Describe the bug**
Recording of Screen using macOS Grab shows two finger touches while rotating map.  Recording with RocketSimApp Pro shows only one finger touch while recording MP4.

**To Reproduce**
1. Launch a Simulator
2. Launch Apple Maps
3. Using the key & mouse strokes:  `<⌥-option>`-`<Mouseclick>`-`<Drag Mouse>`, you can rotate the map.
4. Record using macOS Utility app: Screenshot, which will show two simulated finger touches (left image below)
5. Record using RocketSimApp: _"Start MP4 Recording"_ (right image below)
6. See error

**Expected behavior**
Expect two touches when using two finger gestures in Simulator.

---

**Desktop (please complete the following information):**
- `sw_vers`
> ProductName:		macOS
ProductVersion:		13.0.1
BuildVersion:		22A400

 - RocketSimApp Pro Version `8.7.0 (120)`

---

**Screenshots**

![image](https://user-images.githubusercontent.com/118112/204371233-4a525818-84b1-4d7e-ae1f-7a334caeac00.png)

**Comment by AvdLee** [comment]
@roblabs pinch gestures, yes! That would be great to add for sure 💪

**Comment by roblabs** [comment]
Map recording issue in:  `12.11.15` + iOS 18.1 simulator.   When rotating a map in RocketSim, only a single touch is showing in the recording.



![image](https://github.com/user-attachments/assets/8698d7e3-15b4-45a2-89ea-77302116c9ed)

**Comment by AvdLee** [comment]
@roblabs yes, this is still not fixed. Unfortunately, it's not easy to build. I haven't forgot about it, though!

**Comment by AvdLee** [comment]
This will be available in 15.1.0
https://github.com/user-attachments/assets/f3b899e9-16a2-4383-a63e-0f0ccf3f34a7
